cmd/service/start.go

```diff
@@ -101,21 +135,34 @@ func run(args StartArgs) {
 	// 	fmt.Printf("Starting something #2\n")
 	// }()
 
-	fmt.Printf("Service has started\n")
+	svc.Log.Info("Service has started")
```

Comment:
We can remove those comments above: 
![image](https://github.com/vegaprotocol/vega-monitoring/assets/1239637/7f79b104-459e-4ee6-9325-a9af0804e019)